test.sh

```diff
@@ -22,25 +22,25 @@ do
 done
 
 if [[ -z "$MATCH" && -z "$BLOCK" && -z "$MATCH_TEST" && -z "$MATCH_CONTRACT" ]]; then
-    forge test --use "$SOLC" --fork-url "$ETH_RPC_URL" -vv --force
+    forge test --use "$SOLC" --fork-url "$ETH_RPC_URL" -vv
```

Comment:
This is already corrected on `master` we should probably get it merged here to avoid this change again.

Reply:
merged.